src/Languages/Editor/Impl/Microsoft.Languages.Editor.csproj

```diff
@@ -156,6 +156,10 @@
       <Project>{8d408909-459f-4853-a36c-745118f99869}</Project>
       <Name>Microsoft.Common.Core</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\Windows\Core\Impl\Microsoft.Windows.Core.csproj">
```

Comment:
I kept NS the same since OS-specific part is effectively an extension to the common code so, say, in the package we won't have to have `using ABC` and `using ABC.Windows` all over the place.